libcontainer/cgroups/utils.go

```diff
@@ -217,20 +217,12 @@ func getCgroupMountsHelper(ss map[string]bool, mi io.Reader, all bool) ([]Mount,
 	return res, nil
 }
 
-// GetCgroupMounts returns the mounts for the cgroup subsystems.
-// all indicates whether to return just the first instance or all the mounts.
+// GetCgroupMounts returns the mounts for the cgroup subsystems.  Parameter
+// `all` indicates whether to return just the first instance or all the mounts.
+// This function is cgroupv1 specific and should not be used from cgroupv2 code.
 func GetCgroupMounts(all bool) ([]Mount, error) {
 	if IsCgroup2UnifiedMode() {
-		availableControllers, err := GetAllSubsystems()
-		if err != nil {
-			return nil, err
-		}
-		m := Mount{
-			Mountpoint: unifiedMountpoint,
-			Root:       unifiedMountpoint,
-			Subsystems: availableControllers,
-		}
-		return []Mount{m}, nil
+		return nil, errUnified
```

Comment:
also this change breaks the Kubelet and cAdvisor

Reply:
Fixed